### PR TITLE
include callers in translated Logger metadata for Task

### PR DESCRIPTION
**Pull request by mitchellhenke**
Discussed with @josevalim a bit, and this is to be able to get callers from Logger metadata

Things I'm not sure on: Whether new test should be written, or if there are other modules (GenServer?) that could also do this.

### CODE REVIEW

File: lib/elixir/lib/task/supervised.ex [hunk 1]
```diff
@@ -112,7 +112,8 @@ defmodule Task.Supervised do
           %{
             domain: [:otp, :elixir],
             error_logger: %{tag: :error_msg},
-            report_cb: &__MODULE__.format_report/1
+            report_cb: &__MODULE__.format_report/1,
+            callers: Process.get(:"$callers")
```
Comment (hauleth) [inline lib/elixir/lib/task/supervised.ex:116]: I am not sure if this is public API that we can use there.

**Comment by josevalim** [comment]
This looks great! There are some other tests in this file that say “translates Task”, we should add a similar assertion to them too if possible. Also please rename the test that you currently changed and the one below it to say “:proc_lib+Task”, since we are addi Task based assertions to them. :) thanks!

**Comment by mitchellhenke** [comment]
@josevalim updated!

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**